docs/solutions.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[ecs-solutions]]
+=== Solutions that Support ECS
+
+The following Elastic solutions support ECS out of the box:
+
+* {beats}
```

Comment:
Why are beats in the solution section?
Generally beats are a product or a module, but not a solution by itself.
Solutions include Site Search, App Search, Enterprise Search, SIEM app, Logging UI, etc.

Reply:
I'm using the layman term "solution". This wasn't intended to sound like Elastic solutions which have a specific meaning when talking about Elastic products.

Would the word "tool" be better for this? To me it sounded too low level / technical.

Reply:
And "product" sounds a little bit commercial, given that our tools are all available as open source.

But I think I'll go with "product" for now, until I get feedback.